packages/dmn-js-decision-table/assets/css/dmn-js-decision-table.css

```diff
@@ -426,6 +427,11 @@
   background: var(--view-drd-button-hover-background-color);
 }
 
+/* Adds the missing focus outline to button */
+.dmn-decision-table-container .view-drd .view-drd-button:focus {
+  border: 1px solid var(--view-drd-button-focus-border-color);
+}

```

Comment:
The ViewDrd button had no focus outline comparing with DrillDown button, so I've added one.
But there may be alternative solutions you are considering to add this focus style.